libs/search/src/lib/state/reducer.ts

```diff
@@ -15,6 +15,10 @@ export interface SearchState {
   }
   params: SearchStateParams
   results: {
+    hits: {
+      value: number
+      eq: string
```

Comment:
what is it for ? `eq`

Reply:
"a value of "eq" indicates that "total.value" is the accurate count."  https://www.elastic.co/guide/en/elasticsearch/reference/master/search-your-data.html#track-total-hits